doc/man3/OCSP_resp_find_status.pod

```diff
@@ -100,6 +104,23 @@ OCSP_single_get0_status(). If B<sec> is non-zero it indicates how many seconds
 leeway should be allowed in the check. If B<maxsec> is positive it indicates
 the maximum age of B<thisupd> in seconds.
 
+OCSP_basic_verify() checks that the basic response message B<bs> is correctly
+signed (unless the B<OCSP_NOSIGS> flag is set) and that the signer certificate
+can be validated (unless the <OCSP_NOVERIFY> flag is set). It takes B<st> as the
+trusted store and B<certs> as a set of untrusted intermediate certificates.
+The function tries to find the signer certificate of the response in B<cert>.
+If not found and the B<OCSP_NOINTERN> flag is not set, it also tries any
+certificates the responder may have included in B<bs>.
+It then validates the signer certificate unless it was found in B<certs>
+and the B<OCSP_TRUSTOTHER> flag is set. Otherwise both sources of untrusted
+certificates are used for constructing the validation path for the signer
+certificate unless the B<OCSP_NOCHAIN> flag is set. After successful
+path validation the function immediately yields a positive result if the
+B<OCSP_NOCHECKS> flag is set. Else it verifies that the signer certificate
+meets the OCSP issuer criteria using B<ocsp_check_issuer()>. If the latter
+gives a negative result and the flag B<OCSP_NOEXPLICIT> is not set, the
+function checks for explicit trust for OCSP signing in the root CA certificate.
```

Comment:
I might be tempted to reformulate this as a description of what happens in the default "no flags" case, followed by a list of all the flags explaining how they alter its operation.

Reply:
I see your point and tried this. Doing so would require repeating part of the description of the "no flags" behavior in order to set the context for describing the effect of the respective flags. So I gave up on this as I felt the overall text would become more complicated and significantly longer. Still I was able to streamline the text a bit for better readability.